cypher/cypher-docs/src/docs/dev/ql/constraints/syntax.adoc

```diff
@@ -0,0 +1,143 @@
+[[administration-constraints-syntax]]
```

Comment:
should all of these links be updated to not include `administration`? (also goes for the links in the index and examples files)

Reply:
Yes, I think so, but I think this should be handled in another PR, post publication